linkerd/exp-backoff/src/lib.rs

```diff
@@ -64,6 +63,12 @@ impl ExponentialBackoff {
         if jitter < 0.0 {
             return Err(InvalidBackoff("jitter must not be negative"));
         }
+        if jitter > 100.0 {
+            return Err(InvalidBackoff("jitter must not be greater than 100"));
+        }
+        if !jitter.is_finite() {
+            return Err(InvalidBackoff("jitter must not be finite"));
```

Comment:
i think this is backwards:
```suggestion
            return Err(InvalidBackoff("jitter must be finite"));
```

Reply:
no, the jitter must be finite. this error is thrown when it's not finite.

Reply:
oh i see yes you're right. duh.